NEWS

```diff
@@ -11,9 +11,156 @@
 ### Features:
 ### Bugfixes:
 
-## 1.18.0-rc2 (December 10, 2024)
-### Features: TBD
-### Bugfixes: TBD
+## 1.18.0-rc3 (December 18, 2024)
+### Feature
+#### UCP
+ * Enabled using CUDA staging buffers for pipeline protocols by default
+ * Added endpoint reconfiguration support for non-reused p2p scenarios
+ * Enabled non-cacheable memory domains, activated for GDR
+ * Added user_data parameter to ucp_ep_query
+ * Added support for host memory pipeline through CUDA buffers for rendezvous protocol
+ * Added global VA infrastructure and memory region in absence of error handling
+ * Made protocol performance node names more informative
+ * Enforced always running on the same thread in single thread mode
+ * Multiple improvements in protocols selection infrastructure
+ * Added UCP_MEM_MAP_BLOCK API flag to enforce blocking memory mapping
+ * Allowed up-to 64 endpoint lanes for systems with many transports or devices
+ * Added usage tracker to worker
+ * Improved various logging messages
+#### RDMA CORE (IB, ROCE, etc.)
+ * Added environment variable to manage DC initiator capacity
+ * Added DC dcs_hybrid policy
+ * Reduced MLX5/DV stack size consumption
+ * Added ODP support for verbs and mlx5dv
+ * Added support of CUDA managed memory on IB when ODP is available
+ * Added support of Adaptive Routing on RoCE
+ * Enabled use of implicit ODP with relaxed ordering
+ * Improved GPUDirect detection support on IB
+ * Increased DC initiator default count to 32 for performance optimization
+ * Added ConnectX-8 device support with DDP
+ * Added support for subnet filter list on RoCE GID
+ * Added a reason when a resource's lane is unreachable to request query info string
+ * Added IB MLX5 as a separate UCX module with RPM
+ * Added initial GGA support
+ * Added setting IB DEVX atomic_mode based on device capabilities
+ * Removed DC keepalive
```

Comment:
Removed DC keepalive mechanism, since the keepalive is done on UCP layer

Reply:
fixed